doc/doxygen/src/MainPage.dox

```diff
@@ -6,13 +6,8 @@
  * * 📐 Get off to a good start with the <a
      href="https://github.com/OpenAssetIO/Template-OpenAssetIO-Manager-Python">Python
      manager template.</a>
- * * 📖 See some examples: <a
-     href="https://github.com/ynput/ayon-openassetio-manager-plugin/tree/main#readme">Ayon
-     manager plugin</a>, <a
-     href="https://github.com/OpenAssetIO/usdOpenAssetIOResolver#readme">USD
-     Asset Resolver prototype</a>, <a
-     href="https://github.com/OpenAssetIO/otio-openassetio">OTIO media
-     linker</a>
+ * * 📖 See some  <a href="https://github.com/OpenAssetIO/OpenAssetIO/tree/main/examples#readme">examples</a>
```

Comment:
Nit: Was it intentional to add two spaces after "See some" and the anchor?

Reply:
Good catch. Fixed and took the liberty of amending and force pushing (d9f78a03).